DependencyInjection/Configuration.php

```diff
@@ -197,6 +197,7 @@ private function configureDbalDriverNode(ArrayNodeDefinition $node)
                 ->scalarNode('port')->defaultNull()->end()
                 ->scalarNode('user')->defaultValue('root')->end()
                 ->scalarNode('password')->defaultNull()->end()
+                ->scalarNode('database')->defaultNull()->end()
```

Comment:
We still use in our environment since we actually use databases outside of the default 0

ie.
`snc_client_token_dsn: redis://redist.internalserver.com/5`